cmd/cdi-importer/importer.go

```diff
@@ -255,6 +255,7 @@ func newDataSource(source string, contentType string, volumeMode v1.PersistentVo
 	ep, _ := util.ParseEnvVar(common.ImporterEndpoint, false)
 	acc, _ := util.ParseEnvVar(common.ImporterAccessKeyID, false)
 	sec, _ := util.ParseEnvVar(common.ImporterSecretKey, false)
+	serviceAccount, _ := util.ParseEnvVar(common.ImporterServiceAccountName, false)
```

Comment:
I don't see that this env var is getting set on the import pod.  Here are some reference points:

https://github.com/russellcain/containerized-data-importer/blob/a87f77d4f010ee3fb2ca4b30111b7d70596c2eb2/pkg/controller/import-controller.go#L558
https://github.com/russellcain/containerized-data-importer/blob/a87f77d4f010ee3fb2ca4b30111b7d70596c2eb2/pkg/controller/import-controller.go#L1176

Reply:
ah rats! thanks for closing this gap in my understanding -- let me take a stab at that right now

Reply:
okay so fingers crossed that I understood how this flows in [04d29c3](https://github.com/kubevirt/containerized-data-importer/pull/3433/commits/04d29c38a690926df87a6e2c51794cebb1d7210a)

I dont need to add anything here, right? 
https://github.com/russellcain/containerized-data-importer/blob/41ba02d03ce0205b38fdfc9cbae4a1dc17b3abff/tools/cdi-source-update-poller/main.go#L48
looks like more so this parses env vars (which the SA doesnt fall into) and args passed at invocation (also not applicable)?

should i be adding a check about the source [here](https://github.com/russellcain/containerized-data-importer/blob/a12446110b32c2f3fe1888dd3aed888111a075f6/pkg/controller/import-controller.go#L1290)? and then set it to any value if the PodSpec has a SA set?